src/Test/ElementServices/DRT/Parser/TestClasses/TestClasses.asmmeta

```diff
@@ -0,0 +1,758 @@
+#include "versions.h"
```

Comment:
is this file required? If yes .. MIT license should be added

Reply:
Made the required changes